guides/Live-ImplementationGuide-BaRS/Home/Build/Testing-and-Environments/ReceiverOnboarding.page.md

```diff
@@ -0,0 +1,80 @@
+---
+topic: receiveronboarding
+---
+
+## {{page-title}}
+
+BaRS uses TLS-MA to communicate with Receiving endpoints. Receiving endpoints need a certificate under the NHS Root CA to facilitate TLS-MA.  The receiver needs to follow these steps for Integration and Production environments.
+
+To onboard as a receiver follow these steps:
+
+Step 1: Apply for your domain [apply for a new nhs.uk domain](https://digital.nhs.uk/services/networking-addressing/apply-for-an-nhs.uk-domain-for-websites-and-web-applications).  You must complete Section 5: For website or application records visible on the public internet.
+
+Step 2: Request a certificate under the NHS Root CA. The FQDN must be an nhs.uk address.
+    * There are different certificate chains for INT and PROD
```

Comment:
The detail about cert chains feels wrong here. It's only important once the cert is received.